### PR TITLE
psp-6784: fix missing fields required message displays

### PR DESCRIPTION
**Pull request by buddy326**

### CODE REVIEW

**Comment by buddy326** [comment]
A form component has state because me and Manuel both agreed that it was better than the solution of passing the save function as well as the isValid as props.

**Review by devinleighsmith — CHANGES_REQUESTED** [review]
needs to wait for release.

File: source/frontend/src/features/mapSideBar/acquisition/add/AddAcquisitionForm.tsx [hunk 1]
```diff
@@ -66,6 +66,7 @@ export const AddAcquisitionForm = React.forwardRef<
   };
 
   const handleSubmit = (values: AcquisitionForm, formikHelpers: FormikHelpers<AcquisitionForm>) => {
+    debugger;
```
Comment (devinleighsmith) [inline source/frontend/src/features/mapSideBar/acquisition/add/AddAcquisitionForm.tsx:69]: @buddy326 please remove this.

File: source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/update/UpdateCompensationRequisitionForm.tsx [hunk 2]
```diff
@@ -60,14 +58,13 @@ const UpdateCompensationRequisitionForm: React.FC<CompensationRequisitionFormPro
...
   onCancel,
-  onFooterSave,
-  missingFieldsError,
 }) => {
   const fiscalYearOptions = generateFiscalYearOptions();
   const { setModalContent, setDisplayModal } = useModalContext();
   const formikRef = useRef<FormikProps<CompensationRequisitionFormModel>>(null);
   const [activitiesUpdated, setActivitiesUpdated] = useState<boolean>(false);
   const [showModal, setShowModal] = useState(false);
+  const [isValid, setIsValid] = useState<boolean>(true);
```
Comment (devinleighsmith) [inline source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/update/UpdateCompensationRequisitionForm.tsx:67]: not sure how we still end up with state outside of containers despite my best efforts in code review.

Comment (buddy326) [reply source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/update/UpdateCompensationRequisitionForm.tsx:67]: As I said in my comment above, this was previously in the container but Manuel suggested that in this particular case it was less messy/convoluted to have the isValid variable declared here which I agreed with as well. Before, we were passing isValid and setIsValid as props and it made things less readable. Regardless, I will make the change :)

**Comment by devinleighsmith** [comment]
@buddy326 is this a R4.0 story/bug? if so you'll need to update the PR to target test instead of dev.

**Comment by buddy326** [comment]
@devinleighsmith this is a bug so I will update the PR

**Review by devinleighsmith — APPROVED** [review]

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**